docs/csharp/pattern-matching.md

```diff
@@ -247,7 +247,7 @@ searching for known command values. You might write something like:
 [!code-csharp[VarCaseExpression](../../samples/csharp/PatternMatching/Program.cs#VarCaseExpression "use a var case expression to filter white space")]
 
```

Comment:
In the code file, a comment reads "// whitespace". Could you change it to "// white space"?

Reply:
Done!